string shape_type
geometry_msgs/Point[] key_points
